src/main/java/org/folio/circulation/domain/policy/OverdueFinePolicy.java

```diff
@@ -2,9 +2,49 @@
 
 import io.vertx.core.json.JsonObject;
 
+import java.util.HashMap;
+import java.util.Map;
+
 import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
 
 public class OverdueFinePolicy extends Policy {
+  public enum OverdueFineInterval {
```

Comment:
Why is this interval different to other period intervals, and hence need it's own class definition?

Reply:
This enum represent possible choices for overdue fine interval (f.e. we can fine 5.00 per minute, or 30.00 per month). These intervals are stored in the overdue fine policy records as lowercase strings - "minute", "hour" etc. I didn't find anything in circulation that represents these intervals, hence the new enum.

Reply:
Is it too late to convert the intervals used in the overdue fine policy to use the standard definitions used elsewhere?